stripe/src/test/java/com/stripe/android/view/CardInputWidgetTest.java

```diff
@@ -339,19 +339,24 @@ public void onUpdateText_forAmExPrefix_callsSetImageResourceAndSetsLengthOnCvc()
     @Test
     public void updateToInitialSizes_returnsExpectedValues() {
         // Initial spacing should look like
-        // |---total == 500--------|
-        // |(card==190)--(space==260)--(date=50)|
+        // |img==60||---total == 500--------|
```

Comment:
In Test, I set a value to force the width to be 500 for the framewidth, and the default test dpi creates a width of 60px